skinning/judgement.md

```diff
@@ -0,0 +1,13 @@
+# Judgement
+These are the files that determine what image shows up for each of the judgements in-game. The files are labled as such:
```

Comment:
Maybe mention that they should be in a folder. (Applies to all other sections too)